functions/src/validate-commentable-document.ts

```diff
@@ -1,13 +1,13 @@
 import * as admin from "firebase-admin";
 import * as functions from "firebase-functions";
 import {
-  ICommentableDocumentParams,
-  ICommentableDocumentUnionParams, isCurriculumMetadata, isDocumentMetadata, isWarmUpParams, networkDocumentKey
+  ICommentableDocumentParams, ICommentableDocumentUnionParams, isCurriculumMetadata, isDocumentMetadata,
+  isWarmUpParams, networkDocumentKey
 } from "./shared";
 import { validateUserContext } from "./user-context";
 
 // update this when deploying updates to this function
-const version = "1.0.0";
+const version = "1.0.1";
```

Comment:
While the code in this file didn't change, the imported `isDocumentMetadata()` function did change so we need to redeploy.